diag_manager/diag_manager.F90

```diff
@@ -329,12 +329,12 @@ MODULE diag_manager_mod
      MODULE PROCEDURE send_data_1d
      MODULE PROCEDURE send_data_2d
      MODULE PROCEDURE send_data_3d
-#ifdef OVERLOAD_R8
-     MODULE PROCEDURE send_data_0d_r8
-     MODULE PROCEDURE send_data_1d_r8
-     MODULE PROCEDURE send_data_2d_r8
-     MODULE PROCEDURE send_data_3d_r8
-#endif
+!#ifdef OVERLOAD_R8
+!     MODULE PROCEDURE send_data_0d_r8
+!     MODULE PROCEDURE send_data_1d_r8
+!     MODULE PROCEDURE send_data_2d_r8
+!     MODULE PROCEDURE send_data_3d_r8
+!#endif
```

Comment:
We wanted to check with the GFDL folks regarding removing this OVERLOAD_R8 directive. Specifically, are these codes needed by GFDL for any application support? It seems this change was introduced here: https://github.com/NOAA-GFDL/FMS/commit/70e1c3a2fbffcdb71657a09328cadc34425bab28

Reply:
The reason we commented these out was because of ambiguous `INTERFACE send_data` since `send_data_*d` subroutines now use `class(*)` dummy arguments.

Reply:
@MinsukJi-NOAA I talked with the rest of the team, and we think you should just remove this.  You removed the need for this code, so hopefully it will work.  If not, we will work together to figure out how to make it work.